test/deploy/k8s/cortx-controlnode-statefulset.yml

```diff
@@ -0,0 +1,87 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: controlnode
+  labels:
+    app: controlnode
+spec:
+  serviceName: "controlservice"
+  replicas: 1
+  selector:
+    matchLabels:
+      app: controlnode
+  template:
+    metadata:
+      labels:
+        app: controlnode
+    spec:
+      volumes:
+      - name: cortx-config
+        persistentVolumeClaim:
+          claimName: cortx-config-pvc
+      - name: solution-config
+        configMap:
+          name: solution-config
+      - name: cortx-secret
+        secret:
+          secretName: cortx-secret
+      - name: node-info
+        downwardAPI:
+          items:
+          - path: "name"
+            fieldRef:
+              fieldPath: metadata.name
+      setHostnameAsFQDN: true
```

Comment:
`setHostnameAsFQDN: true` will not work with our standard kubectl version v1.19
Lets set `hostname` in container directly by copying value from `hostname --fqdn`